concordium-sdk-examples/src/main/java/com/concordium/sdk/examples/contractexample/cis2nft/Cis2Nft.java

```diff
@@ -130,6 +133,20 @@ private void handleInit(ClientV2 client, Nonce nonce) {
     }
 
     private void handleMethod(ClientV2 client, Nonce nonce, SchemaParameter parameter) {
+
+        // Estimate energy used by transaction.
+        InvokeInstanceRequest invokeInstanceRequest = InvokeInstanceRequest.from(BlockQuery.LAST_FINAL,
+                CONTRACT_ADDRESS,
+                parameter,
+                Optional.empty());
+        InvokeInstanceResult invokeInstanceResult = client.invokeInstance(invokeInstanceRequest);
+        Energy usedEnergy = invokeInstanceResult.getUsedEnergy();
+        ChainParameters parameters = client.getChainParameters(BlockQuery.LAST_FINAL);
+        // Convert to Euro and CCD using ChainParameters from the best block and utility methods in the Converter class.
+        BigDecimal euros = CurrencyConverter.energyToEuro(usedEnergy, parameters).asBigDecimal(6);
```

Comment:
Why 6 decimals for EUR?

Reply:
Just an arbitrary number, maybe 2 is more fitting?

Reply:
That is what I expected. But I guess it's an example, so either is fine.

Reply:
You risk getting 0 if it's just 2 decimals :)